website/docs/02-separation-logic-and-biabduction.md

```diff
@@ -47,7 +47,7 @@ formula above can be visualized like so:
 
 ![](/img/SepSplit.jpg)
 
-The important thing about separating conjunction is the way that it fits
+The important thing about separating conjunctions is the way that it fits
```

Comment:
The singular was correct here (referred to by "it" right after). Maybe "The important thing about *the* separating conjunction is ..."?